wire-ios/Wire-iOS Tests/ConversationMessageCell/MessageActionsViewControllerTests.swift

```diff
@@ -17,13 +17,16 @@
 //
 
 import SnapshotTesting
-@testable import Wire
+import WireCommonComponents
 import XCTest
 
+@testable import Wire
+
 final class MessageActionsViewControllerTests: XCTestCase {
 
     override func setUp() {
         super.setUp()
+        FontScheme.configure(with: .large)
```

Comment:
We have used `BaseSnapshotTestCase` for this so far, right?
Why did you choose not to subclass now?

Anyway, I actually prefer writing this line in every class instead of subclassing.